core/tests/protractor_desktop/explorationTranslationTab.js

```diff
@@ -153,71 +179,113 @@ describe('Exploration translation', function() {
-      explorationEditorSettingsTab.setTitle('Check');
-      explorationEditorSettingsTab.setCategory('Algorithms');
-      explorationEditorSettingsTab.setObjective('To check the translation tab');
-      explorationEditorPage.navigateToMainTab();
-      explorationEditorMainTab.setStateName('one');
-      explorationEditorMainTab.setContent(forms.toRichText(
-        'This is first card.'));
-      explorationEditorMainTab.setInteraction('NumericInput');
-      explorationEditorMainTab.addResponse(
-        'NumericInput', forms.toRichText('This is feedback1.'),
-        'two', true, 'Equals', 6);
-      var responseEditor = explorationEditorMainTab.getResponseEditor(
-        'default');
-      responseEditor.setFeedback(forms.toRichText(
-        'This is default_outcome.'));
-      explorationEditorMainTab.addHint('This is hint1.');
-      explorationEditorMainTab.addHint('This is hint2.');
-      explorationEditorMainTab.addSolution('NumericInput', {
-        correctAnswer: 6,
-        explanation: 'This is solution.'
-      });
-      explorationEditorMainTab.moveToState('two');
-      explorationEditorMainTab.setContent(forms.toRichText(
-        'This is second card.'));
-      explorationEditorMainTab.setInteraction('NumericInput');
-      explorationEditorMainTab.addResponse(
-        'NumericInput', forms.toRichText('This is feedback1.'),
-        'final card', true, 'Equals', 7);
-      responseEditor = explorationEditorMainTab.getResponseEditor(
-        'default');
-      responseEditor.setFeedback(forms.toRichText('This is default_outcome.'));
-      explorationEditorMainTab.addHint('This is hint1.');
-      explorationEditorMainTab.addHint('This is hint2.');
-      explorationEditorMainTab.addSolution('NumericInput', {
-        correctAnswer: 7,
-        explanation: 'This is solution.'
-      });
-      explorationEditorMainTab.moveToState('final card');
-      explorationEditorMainTab.setInteraction('EndExploration');
-      explorationEditorMainTab.moveToState('two');
+      users.login('user@editorTab.com');
+      creatorDashboardPage.get();
+      creatorDashboardPage.editExploration('Test Exploration');
       explorationEditorPage.navigateToTranslationTab();
       explorationEditorTranslationTab.exitTutorial();
+      explorationEditorTranslationTab.changeLanguage('Hindi');
+      explorationEditorTranslationTab.switchToTranslationMode();
       explorationEditorTranslationTab.navigateToFeedbackTab();
-      explorationEditorPage.saveChanges();
+      explorationEditorTranslationTab.setTranslation(forms.toRichText(
+        'Sample Translation.'));
+      explorationEditorPage.saveChanges('Adds one translation.');
       explorationEditorTranslationTab.expectFeedbackTabToBeActive();
       workflow.publishExploration();
       explorationEditorTranslationTab.expectFeedbackTabToBeActive();
+      users.logout();
     });
 
 
   it('should change translation language correctly', function() {
```

Comment:
Maybe we can club the default mode check with this one? (As these aren't unit tests, I think we can combine 2 tests that essentially test the same behavior. Happy to leave it as separate tests too. What do you think?

Reply:
Oops: Meant this comment to be on 'should correctly switch to different modes' block